src/Components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  deleteTodo?: (todoId: number) => void;
+  selectedId: number;
+  isLoadingCompleted?: boolean;
+  isLoading?: boolean;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  deleteTodo,
+  selectedId,
+  isLoadingCompleted,
+  isLoading,
```

Comment:
currently, you use it only for tempTodo
But you may set the id of `tempTodo` (0) to `selectedId` and get rid of `isLoading` prop here